shims/common/src/main/scala/org/apache/gluten/GlutenConfig.scala

```diff
@@ -1962,6 +1966,17 @@ object GlutenConfig {
       .booleanConf
       .createWithDefault(true)
 
+  val COST_EVALUATOR_ENABLED =
+    buildConf(GlutenConfig.GLUTEN_COST_EVALUATOR_ENABLED)
+      .internal()
+      .doc(
+        "If true and gluten enabled, use " +
+          "org.apache.spark.sql.execution.adaptive.GlutenCostEvaluator as custom cost " +
+          "evaluator class, else follow the configuration " +
+          "org.apache.spark.sql.execution.adaptive.GlutenCostEvaluator.")
```

Comment:
`org.apache.spark.sql.execution.adaptive.GlutenCostEvaluator` this should be corrected

Reply:
My mistake, fixed, thank you.